packages/components/atoms/f-segmented-control/src/components/SegmentedControl.vue

```diff
@@ -0,0 +1,254 @@
+        }
+    },
+    data () {
+        const selectedIndex = this.options.findIndex(option => option.selected);
+
+        return {
+            selectedLabel: (selectedIndex !== undefined && this.options[selectedIndex])
+                ? this.options[selectedIndex].label
+                : this.options[0].label,
+            tabIndex: selectedIndex > -1 ? selectedIndex : 0
+        };
+    },
+    computed: {
+        // dynamically set the tabindex attribute on each button, giving focus only to the currently selected button
+        focusedTabIndex () {
+            return this.options.map((_, index) => (index === this.tabIndex ? 0 : -1));
+        }
+    },
+    methods: {
+        selectOption (label) {
+            const previousIndex = this.tabIndex;
+            const newIndex = this.options.findIndex(option => option.label === label);
+
+            this.selectedLabel = label;
+
+            this.$emit('input', label);
+
+            if (newIndex !== previousIndex) {
+                this.focusNewButton(newIndex, this.$el.children[previousIndex]);
+            }
+        },
+        onKeyDown (event) {
+            let newIndex = this.tabIndex;
+            const currentButton = event.target;
+
+            // We're using the roving tabindex pattern here:
+            // Only the focused element has tabindex not set to -1,
+            // so users can easily return to their last focused item when tabbing back.
+            if (event.key === 'ArrowRight' || event.key === 'ArrowDown') {
+                event.preventDefault();
+                // Move focus to the next button
+                newIndex = (this.tabIndex + 1) % this.options.length;
+            } else if (event.key === 'ArrowLeft' || event.key === 'ArrowUp') {
+                event.preventDefault();
+                // Move focus to the previous button
+                newIndex = (this.tabIndex - 1 + this.options.length) % this.options.length;
+            } else if (event.key === 'Enter' || event.key === 'Space') {
+                event.preventDefault();
+                this.selectOption(this.options[this.tabIndex].label);
+
+                return;
+            }
+
+            // If the newIndex has changed, update the tabIndex and focus the new button
+            if (newIndex !== this.tabIndex) {
+                this.focusNewButton(newIndex, currentButton);
+            }
+        },
+        focusNewButton (newIndex, currentButton) {
+            this.tabIndex = newIndex;
+            // Remove focus from the current button
+            currentButton.blur();
+
+            // Wait for the DOM to update, then focus the new button
+            this.$nextTick(() => {
+                this.$el.children[this.tabIndex].focus();
+            });
+        }
+    }
+};
+</script>
+
+<style lang="scss" module>
+@use '@justeat/fozzie/src/scss/fozzie' as f;
+
+// Variables
+$sc-sm-height: f.spacing(f);
+$sc-l-height: calc(f.spacing(b) * 6);
```

Comment:
Super readability NIT:

```suggestion
$sc-height-small: f.spacing(f);
$sc-height-large: calc(f.spacing(b) * 6);
```